### ISSUE TITLE
replace lib log to logrus or other logging library

### ISSUE THREAD

**Issue by cpanato** [falcosecurity/falcosidekick#140]
**Motivation**
using another logging library like logrus we can use some features, like enable log level and other things and not need to write the type of the log by hand as we do today.

since this can be a breaking change for people that are parsing the logs and the format will change a bit we can make a major release

/milestone 3.0.0

**Comment by gauravgahlot** [comment]
@cpanato  @Issif Can I take this up if not taken already?

**Comment by Issif** [comment]
@gauravgahlot This is not already taken but I wanted to have it (from me or else) for v3 release. It may change the log format and break indexation of logs for customers, I want to avoid that without bumping version. I don't know when we'll start on v3, I started to write a paper about I can share with everybody soon.

**Comment by gauravgahlot** [comment]
@Issif Okay. I will be happy to help anyway I can. So, please do let me know.

**Comment by bdashrad** [comment]
This still would be great to have. Needing to write a custom parser to not have all of the logs show as error is not a great experience.

**Comment by Issif** [comment]
@bdashrad I agree, but it implies a lot of changes, we'll see that point in future v3
